src/state/slices/opportunitiesSlice/resolvers/uniV2/index.ts

```diff
@@ -291,9 +284,6 @@ export const uniV2LpLpOpportunityIdsResolver = ({
 }> => {
   const { getState } = reduxApi
   const state: any = getState()
-  const { DynamicLpAssets } = selectFeatureFlags(state)
-
-  if (!DynamicLpAssets) return Promise.resolve({ data: [...foxEthLpAssetIds] })
```

Comment:
We do not use this flag anymore

Reply:
Actually disregard me - we still want this https://github.com/shapeshift/web/pull/4800